llvm/lib/Target/RISCV/RISCVISelLowering.cpp

```diff
@@ -14225,23 +14259,6 @@ SDValue RISCVTargetLowering::PerformDAGCombine(SDNode *N,
         return DAG.getConstant(-1, DL, VT);
       return DAG.getConstant(0, DL, VT);
     }
-    case Intrinsic::riscv_vloxei:
```

Comment:
To draw attention here.  I removed the previous transform over the riscv specific nodes.  I could leave it here if desired, I'm just not sure that's worthwhile.  The result of removing is some test regression in CodeGen/RISCV/rvv/narrow-shift-extend.ll, but I'm not sure if this regression "matters" or if this is synthetic test coverage.  Reverting this piece of the change is simple, I'm happy to go with reviewer preference here.